php/PHPCD.php

```diff
@@ -989,16 +989,19 @@ public function classmap($pattern)
         $use_fqdn = isset($pattern[0]) && $pattern[0] === "\\";
         $pattern = trim($pattern, "\\");
         $classmap_file = $this->root.'/vendor/composer/autoload_classmap.php';
-        $classmap = get_declared_traits() + get_declared_interfaces()
-            + get_declared_classes();
+        $declaredSymbols = array_merge(get_declared_traits(), get_declared_interfaces(),
```

Comment:
what's the difference between array_merge and array sum?

Reply:
The difference is vital on numeric keys arrays (like what `get_declared_clasases()` and others returns).
Here's an example:  https://3v4l.org/TVDbX